.circleci/config.yml

```diff
@@ -182,7 +182,7 @@ jobs:
           name: Ensure translations are not missing
           command: |
             set -euo pipefail
-            diff <(cat packages/mobile/locales/en-US/*.json | jq keys | sort) <(cat packages/mobile/locales/es-AR/*.json | jq keys | sort)
+            diff <(cat packages/mobile/locales/es-AR/*.json | jq -r '[paths | join(".")][]' | sort) <(cat packages/mobile/locales/en-US/*.json | jq -r '[paths | join(".")][]' | sort)
```

Comment:
What do you think of moving this to a script in `mobile/scrips` and exposing it via a `yarn` command?
That way we can run this test locally if we want and also we don't have this big scary one-liner here

Reply:
Sounds like a good idea, however `jq` is an additional dependency that devs would need to install.  Would that be okay?

Reply:
That's okay. Please have the script output a helpful error message if jq is not installed :)